src/setupTests.js

```diff
@@ -0,0 +1,11 @@
+import httpAdapter from 'axios/lib/adapters/http';
+import httpClient from 'services/httpClient';
```

Comment:
```suggestion

import httpClient from 'services/httpClient';
```

Reply:
why?

Reply:
We are always leaving an empty line between the imports of files that are ours and externals, right? 🤔

Reply:
Ok. Didn't notice that. I thought it only was for absolute/relative imports